docs/amqp.md

```diff
@@ -291,11 +303,13 @@ Modifying message annotations with the [modified](https://docs.oasis-open.org/am
 Modifying a message in a [stream](./streams) doesn't make sense given that a stream is an immutable log.
 
 If field `undeliverable-here` is
-* `true`, classic queues and quorum queues will [dead letter](/docs/dlx) the message. If dead lettering is not configured, the message will be discarded.
+
+* `true`, classic queues and quorum queues will [dead letter](./dlx) the message. If dead lettering is not configured, the message will be discarded.
 * `false`, classic queues and quorum queues will requeue the message.
 
 :::warning
+The behaviour of `undeliverable-here` may change in a future RabbitMQ version.
```

Comment:
@michaelklishin as explained in https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier can we please make sure to keep the empty lines around the contents when using admonitions?

Reply:
I opened https://github.com/rabbitmq/rabbitmq-website/pull/2043 which adds the empty lines back.

Reply:
They are not used in most if not all other guides. If we decide to adopt Prettier, it must be done to every admonition there is. I am not aware of anyone using that tool.